cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -255,3 +318,24 @@ Previously released parcels can also be accessed from their version-specific URL
   |http:|//repository.cask.co/parcels/cdap/2.8/CDAP-2.8.0-1-precise.parcel
   |http:|//repository.cask.co/parcels/cdap/2.8/CDAP-2.8.0-1-trusty.parcel
   |http:|//repository.cask.co/parcels/cdap/2.8/CDAP-2.8.0-1-wheezy.parcel
+  
+
+.. _cloudera-troubleshooting-upgrade-cdh:
+
+.. rubric:: Problems While Upgrading CDH
+
+If you miss a step in the upgrade process and something goes wrong, it's possible that the
+tables will get re-enabled before the coprocessors are upgraded. This could cause the
+regionservers to abort and may make it very difficult to get the cluster back to a stable
+state where the tables can be disabled again and complete the process.
+
+In that case, set this configuration property in ``hbase-site.xml``::
+
+  hbase.coprocessor.abortonerror = false
```

Comment:
Should we quote the actual text that would be added to the file here?

<property>
  <name>hbase.coprocessor.abortonerror</name>
  <value>false</value>
</property>

I'm not sure how we normally display the config properties, but this would be what actually needs to be added.